main.go

```diff
@@ -536,6 +541,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controllers.PostgreSQLUserReconciler{
+		Reconciler: &controllers.AsyncReconciler{
+			Client:      mgr.GetClient(),
+			AzureClient: psqlusermanager,
+			Telemetry: telemetry.InitializeTelemetryDefault(
+				"PSQLUser",
+				ctrl.Log.WithName("controllers").WithName("PSQLUser"),
+			),
+			Recorder: mgr.GetEventRecorderFor("PSQLUser-controller"),
```

Comment:
We should name this `PostgreSQLUser` to stick with naming conventions, since that is the name inside of our controller!

Similar to how the rest of the PostgreSQL resources are declared